packages/mds-attachment-service/repository/index.ts

```diff
@@ -1,11 +1,56 @@
-import { ReadWriteRepository } from '@mds-core/mds-repository'
+import { DeleteReturning, InsertReturning, ReadWriteRepository, RepositoryError } from '@mds-core/mds-repository'
+import { AttachmentDomainToEntityCreate, AttachmentEntityToDomain } from './mappers'
+import { AttachmentDomainModel } from '../@types'
+import { AttachmentEntity } from './entities/attachment-entity'
+
 import entities from './entities'
 import migrations from './migrations'
 
 class AttachmentReadWriteRepository extends ReadWriteRepository {
   constructor() {
     super('attachments', { entities, migrations })
   }
+
+  public writeAttachment = async (attachment: AttachmentDomainModel): Promise<AttachmentDomainModel> => {
+    const { connect } = this
```

Comment:
I started moving away from this in favor of just `const connection = await this.connect('rw')`